.gitignore

```diff
@@ -23,3 +23,6 @@ cairo-vm-cli/air_input.pub
 ensure-no_std/Cargo.lock
 
 !vm/src/tests/cairo_pie_test_output.json
+
+# JetBrains IDE files
+.idea
```

Comment:
Not sure this is welcome upstream. Maybe try `git config --global core.excludesFile ".idea"` instead?

Reply:
Why wouldn't it be welcome upstream?

Reply:
pollution and/or merge conflicts. no biggie, but I do think the `git config --global` above might prevent you from having to add it to each project you work on